Author: Alastair McKinstry  <mckinstry@debian.org>
Description: Patch needed for UCHAR_MAX defintion on mips, powerpc and other
 archs.
Last-Updated: 2014-07-08
Forwarded: no

Index: cdo-1.6.4+dfsg.1/libcdi/src/cksum.c
===================================================================
--- cdo-1.6.4+dfsg.1.orig/libcdi/src/cksum.c
+++ cdo-1.6.4+dfsg.1/libcdi/src/cksum.c
@@ -5,6 +5,7 @@
 
 #include <stdlib.h>
 #include <sys/types.h>
+#include <limits.h>
 
 static const uint32_t crctab[] = {
   0x00000000,
